js/bootstrap-select.js

```diff
@@ -2113,13 +2113,11 @@
       if (doneButton) menu.appendChild(doneButton);
       newElement.appendChild(menu);
 
-      var $li = $(li);
-      $("a", $li).show(); // always make sure first item is shown, otherwise offsetHeight = 0
-      var $a = $("a", $li);
-      var itemIsBlank = $a.text() == "";
-      if (itemIsBlank) {
+      var newElementAnchor = $('a', $(li));
+      newElementAnchor.show(); // always make sure first item is shown, otherwise offsetHeight = 0
+      if (newElementAnchor.text() == '') {
```

Comment:
Use `===` here. I will add a proper eslint config at some point for this kind of stuff.